.github/workflows/python-test.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       # You can use PyPy versions in python-version.
       # For example, pypy2 and pypy3
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
do we keep 3.6?

Reply:
Did you see the first line of the commit message?

Reply:
Any reasons that we want to drop it @cclauss ? I think we probably should keep 3.6 as some of people might still use it, do you mind add it back? Thanks!